packages/millicast-multiview-demo/src/multiviewer.js

```diff
@@ -29,37 +38,29 @@ let sourceTracksMap = {}
 // This will store a mapping: transceiver video mid => source id
 let transceiverMidToSourceIdMap = {}
 
-// This will store a mapping: transceiver video mid => active layers 
+// This will store a mapping: transceiver video mid => active layers
 let transceiverToLayersMap = {}
 
 // Create a new viewer instance
-const tokenGenerator = () => Director.getSubscriber(streamName, accountId)
+const tokenGenerator = () => Director.getSubscriber(streamName, accountId, subscriberToken, enableDRM)
 let viewer
 
-
-viewer.on('metadata', (metadata) => {
-  console.log(`Metadata event from ${transceiverToSourceIdMap[metadata.mid] || 'main'}:`, metadata)
-})
-
 document.addEventListener('DOMContentLoaded', async () => {
   remoteVideosContainer = document.getElementById('remoteVideos')
   mainVideoElement = document.getElementById('mid-0')
   mainAudioElement = document.getElementById('mid-1')
-  console.log(' page loaded, mainVideoElement is', mainVideoElement)
   try {
     viewer = new View(streamName, tokenGenerator)
+    viewer.on('onMetadata', (metadata) => {
```

Comment:
nitpick non blocking : The next version of the SDK will not report `onMetadata`. Instead it will only be `metadata`. You can use that instead

Reply:
will change it